src/core/index.ts

```diff
@@ -167,41 +254,56 @@ export const dismissAllApprovals = async (context: Context): Promise<void> => {
  * opens the pull request.
  *
  * @param context The context for the webhook request constructed by Probot core.
+ * @param failOnPurpose Make the run fail on purpose. For testing purposes only.
  */
-export const maybeApproveChange = async (context: Context): Promise<void> => {
+export const maybeApproveChange = async (
+  context: Context,
+  failOnPurpose = false,
+  error: Error | undefined = undefined,
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6092518)